FlowCryptUI/Nodes/CustomAlertNode.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  CustomAlertNode.swift
+//  FlowCrypt
+//
+//  Created by Ioan Moldovan on 12/8/24
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import AsyncDisplayKit
+
+public final class CustomAlertNode: CoreAlertNode, ASTextNodeDelegate {
+
+    private var overlayNode: ASDisplayNode!
+    private var contentView: ASDisplayNode!
+    private var separatorNode: ASDisplayNode!
+    private var titleLabel: ASTextNode!
+    private var messageLabel: ASTextNode!
+    private var okayButton: ASButtonNode!
+
+    private let title: String
+    private let message: String?
+
+    public var onOkay: (() -> Void)?
+
+    // MARK: - Initialization
+
+    public init(
+        title: String,
+        message: String? = nil
+    ) {
+        self.title = title
+        self.message = message
+        super.init()
+        setupNodes()
+    }
+
+    private func createNodes() {
+        overlayNode = createOverlayNode()
+        contentView = createContentView()
+        separatorNode = createSeparatorNode()
+        titleLabel = createTextNode(text: title, isBold: true, fontSize: 17)
+        messageLabel = createTextNode(text: message ?? "", isBold: false, fontSize: 13, identifier: "aid-custom-alert-message", detectLinks: true)
```

Comment:
let's center message text, to make it look like in default iOS alerts:

<img width="300" alt="Screenshot 2024-12-11 at 12 47 15" src="https://github.com/user-attachments/assets/9da05649-cbfb-4c42-ac08-98f2a65b5e6e">

<img width="254" alt="Screenshot 2024-12-11 at 12 46 54" src="https://github.com/user-attachments/assets/90c88e7b-b746-4a0f-9106-1aa51c7f2f06">

and is it possible to highlight links by different text color, not by underlining?
it seems to be more common on iOS

Reply:
Okay, got it